### ISSUE TITLE
AttributeError: module 'streamlit' has no attribute 'fragment'

### ISSUE THREAD

**Issue by amirbtb** [whitphx/stlite#1079]
Hello,

It seems that the `@st.fragment` decorator (documented [here](https://docs.streamlit.io/develop/api-reference/execution-flow/st.fragment)) isn't available in stlite.

Here is my `index.html` file : 
```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1, shrink-to-fit=no"
    />
    <title>stlite app</title>
    <link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/@stlite/mountable@0.63.1/build/stlite.css"
    />
  </head>
  <body>
    <div id="root"></div>
    <script src="https://cdn.jsdelivr.net/npm/@stlite/mountable@0.63.1/build/stlite.js"></script>
    <script>
        stlite.mount(
            {
                entrypoint: "main.py",
                files: {
                    "main.py": {
                        url: "./main.py",
                    },
                    "file1.py": {
                        url: "./file1.py",
                    },
                    "file2.py": {
                        url: "./file2.py",
                    },
                    "file3.py": {
                        url: "./file3.py"
                    },
                    "database.db": {
                        url: "../database/database.db"
                    }
                }
            }
        );
    </script>
  </body>
</html>
```

Here is my call to `@st.fragment` in `file1.py` : 

```python
import streamlit as st

st.header('a section')

@st.fragment
def add_container():
    container = st.container(border=True)
    container.write("This is inside the container")


if st.button('Add container'):
    add_container()
```

Content of the `main.py` file : 
```python
import micropip
await micropip.install(['sqlite3', "sqlalchemy"])

import streamlit as st

st.title('My App')

pg = st.navigation([st.Page("file1.py"), st.Page("file2.py")])
pg.run()
```

When serving the `index.html` file, I get the following error : 
```
AttributeError: module 'streamlit' has no attribute 'fragment'
Traceback:
File "streamlit/runtime/scriptrunner/script_runner.py", line 596, in _run_script
File "/home/pyodide/main.py", line 9, in <module>
    pg.run()
File "streamlit/navigation/page.py", line 286, in run
File "/home/pyodide/file1.py", line 5, in <module>
    @st.fragment
     ^^^^^^^^^^^
```
Thank you again for this amazing work and for your time looking at this issue 🙏🏽

**Comment by whitphx** [comment]
The latest Stlite version, 0.63.1 is based on Streamlit 1.36.0, and it exposes the fragment feature via `st.experimental_fragment`, not `st.fragment`.
Plz be patient until the next release of Stlite that will be based on Streamlit>=1.37.0 with de-experimentalized `st.fragment`.

**Comment by amirbtb** [comment]
Thank you very much I just tried `st.experimental_fragment`and it works fine.
Will follow your releases !

**Comment by whitphx** [comment]
Stlite 0.64.0 was released and it's based on Streamlit 1.37.1 👍
